### PR TITLE
Wrapper for autoai_libs.nsfa transformer added

### PR DESCRIPTION
**Pull request by Rafalll-Maciasz**
## Changes:
 - schema and wrapper for new autoai_libs transformer NSFA

### CODE REVIEW

**Review by shinnar — CHANGES_REQUESTED** [review]
I left a review on the previous PR https://github.com/IBM/lale/pull/1329 (which was closed in favor of this one).  Please take a look at the comments there

**Comment by Rafalll-Maciasz** [comment]
You are right, input/output for fit and transform schema are changed to the `number` type.

File: lale/lib/autoai_libs/__init__.py [hunk 1]
```diff
@@ -102,6 +104,7 @@
 from .float_str2_float import FloatStr2Float as FloatStr2Float
 from .fs1 import FS1 as FS1
 from .fs2 import FS2 as FS2
+from .nsfa import NSFA as NSFA
```
Comment (shinnar) [inline lale/lib/autoai_libs/__init__.py:107]: In order to solve the import error that results from when the lale.autai_libs module is imported, but the version of autai_libs being used does not have NSFA (as is happening now), this import should be behind a version gate.
An example of getting the version and testing it is here:
https://github.com/IBM/lale/blob/09f4c84140fc45a2488f405130fcf29831fe19bc/lale/lib/autoai_libs/cat_imputer.py#L159

**Review by shinnar — CHANGES_REQUESTED** [review]
Thanks for the fixes.  There is a remaining problem, due to these schemas being hosted in lale instead of in the same repository as the class they are wrapping

**Comment by shinnar** [comment]
Hi @Rafalll-Maciasz .  any updates on this PR?  As noted in the last review, it needs one minor change (putting back the simple wrapper impl class you had before, sorry about that), and I would then be happy to merge it.

**Comment by Rafalll-Maciasz** [comment]
Hi @shinnar, no problem, I have added the impl wrapper class back.

**Pull request merged**